Client/Application/AppDelegate.swift

```diff
@@ -291,8 +304,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func launchFromURL(params: LaunchParams) {
         let isPrivate = params.isPrivate ?? false
+        let fxaParams = params.fxaParams ?? nil
```

Comment:
I think it'll be simpler if you just make a new method instead of using the existing LaunchFromURL. That way you dont need to check for a valid fxaParams here.